src/utils/looper.ts

```diff
@@ -1,76 +1,78 @@
 
-      const serverStatuses = processProbeStatus({
-        probe: item,
-        validatedResp,
-        trueThreshold: item.trueThreshold || 5,
-        falseThreshold: item.falseThreshold || 5,
-      })
+    const serverStatuses = processProbeStatus({
+      probe,
+      validatedResp,
+      trueThreshold: probe.trueThreshold || 5,
+      falseThreshold: probe.falseThreshold || 5,
+    })
 
-      serverStatuses.forEach(async (status, index) => {
-        if (status.shouldSendNotification) {
-          log(`Sending a "${item.alerts[index]}" notification`)
-          await sendAlerts({
-            validation: validatedResp[index],
-            notifications: config.notifications,
-            url: item.request.url ?? '',
-            status: status.isDown ? 'DOWN' : 'UP',
-          })
-        }
-      })
-    } catch (error) {
-      log(
-        'id:',
-        item.id,
-        '- status: ERROR for:',
-        item.request.url,
-        ':',
-        error.message
-      )
-    }
-  })
+    serverStatuses.forEach(async (status, index) => {
+      if (status.shouldSendNotification) {
+        log(`Sending a "${probe.alerts[index]}" notification`)
+        await sendAlerts({
+          validation: validatedResp[index],
+          notifications: notifications,
+          url: probe.request.url ?? '',
+          status: status.isDown ? 'DOWN' : 'UP',
+        })
+      }
+    })
+  } catch (error) {
+    log(
+      'id:',
+      probe.id,
+      '- status: ERROR for:',
+      probe.request.url,
+      ':',
+      error.message
+    )
+  }
 }
 
 export function looper(config: Config) {
-  const interval = config.interval ?? 0
-
   log('Probes:')
-  config.probes.forEach(async (item) => {
-    log(`Probe ID: ${item.id}`)
-    log(`Probe Name: ${item.name}`)
-    log(`Probe Description: ${item.description}`)
-    log(`Probe Request Method: ${item.request.method}`)
-    log(`Probe Request URL: ${item.request.url}`)
-    log(`Probe Request Headers: ${JSON.stringify(item.request.headers)}`)
-    log(`Probe Request Body: ${JSON.stringify(item.request.body)}`)
-    log(`Probe Alerts: ${item.alerts.toString()}\n`)
-  })
 
-  doProbes(config).catch((error) => console.error(error.message))
-  if (interval > 0) {
-    setInterval(doProbes, interval * MILLISECONDS, config)
-  }
+  config.probes.forEach(async (probe) => {
+    log(`Probe ID: ${probe.id}`)
+    log(`Probe Name: ${probe.name}`)
+    log(`Probe Description: ${probe.description}`)
+    log(`Probe Interval: ${probe.interval}`)
+    log(`Probe Request Method: ${probe.request.method}`)
+    log(`Probe Request URL: ${probe.request.url}`)
+    log(`Probe Request Headers: ${JSON.stringify(probe.request.headers)}`)
+    log(`Probe Request Body: ${JSON.stringify(probe.request.body)}`)
+    log(`Probe Alerts: ${probe.alerts.toString()}\n`)
+
+    const probeInterval = setInterval(async () => {
+      return doProbe(probe, config.notifications)
+    }, (probe.interval ?? 0) * MILLISECONDS)
```

Comment:
I think default value is better be 10 seconds. Please put the default value as const at the beginning of the file.